enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk_setup/components/SdkIframeEmbedPreview.tsx

```diff
@@ -32,20 +33,26 @@ export const SdkIframeEmbedPreview = () => {
   const instanceUrl = useSetting("site-url");
   const applicationColors = useSetting("application-colors");
 
-  // TODO(EMB-696): There is a bug in the SDK where if we set the theme back to undefined,
-  // some color will not be reset to the default (e.g. text color, CSS variables).
-  // We can remove this block once EMB-696 is fixed.
-  const defaultTheme: MetabaseTheme = useMemo(() => {
-    const colors = Object.fromEntries(
-      getConfigurableThemeColors().map((color) => [
-        color.key,
-        applicationColors?.[color.originalColorKey] ??
-          defaultMetabaseColors[color.originalColorKey],
-      ]),
+  // Apply derived colors to the theme for the preview
+  const derivedTheme = useMemo(() => {
+    // TODO(EMB-696): There is a bug in the SDK where if we set the theme back to undefined,
```

Comment:
I just moved the defaultTheme inside the derivedTheme block so they can be memoized together.